src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -3,18 +3,139 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { Tooltip as TooltipComponent, TooltipProps } from "./Tooltip";
+import { BrightHeaderVariants } from "./partials/BrightHeader";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
 
 export default {
     title: "Components/Tooltip",
     component: TooltipComponent,
     args: {
-        tooltip: "Tooltip Content (Reactnode)",
+        tooltip: "Your text here",
+        heading: "",
+        buttons: [],
+    },
+    argTypes: {
+        brightHeader: {
+            options: [
+                null,
+                BrightHeaderVariants.Information,
+                BrightHeaderVariants.Warning,
+                BrightHeaderVariants.Tip,
+                BrightHeaderVariants.Note,
+            ],
+            control: { type: "select" },
+        },
+        tooltipIcon: {
+            options: ["Nothing", "Icon"],
+            mapping: {
+                Nothing: null,
+                Icon: <IconIcons key="1" />,
+            },
+            control: { type: "select" },
+        },
+        headingIcon: {
+            options: ["Nothing", "Icon"],
+            mapping: {
+                Nothing: null,
+                Icon: <IconIcons key="1" />,
+            },
+            control: { type: "select" },
+        },
     },
 } as Meta;
 
-export const Tooltip: Story<TooltipProps> = (args: TooltipProps) => (
-    <div>
-        <TooltipComponent {...args} />
-        <p>Some other text.</p>
-    </div>
-);
+const loremIpsum = "Cupcake ipsum dolor sit amet ice cream.";
+
+export const TooltipTemplate: Story<TooltipProps> = (args: TooltipProps) => <TooltipComponent {...args} />;
+
+export const Default = TooltipTemplate.bind({});
+Default.args = {
+    brightHeader: undefined,
```

Comment:
You don't have to set this explicitly if it's not needed. Same for the `buttons` control below.